agora-drep-spec/Spec/Proxy/Context.hs

```diff
@@ -209,6 +217,7 @@ validGAT3Mint config =
       [ withMinting (gat3CurSym config)
       , input (gat2Utxo config)
       , mint (Value.singleton (gat3CurSym config) (TokenName "") 1)
+      , mint (Value.singleton gat2CurSym (TokenName "") (-1))
```

Comment:
Added these cause Minting and Spending is ran always in the same transaction and optimizations depend on the shape of `mint` `PValue`